<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RessourcesBrowser</class>
 <widget class="QWidget" name="RessourcesBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="lblDb">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="icon">
       <iconset resource="../../../ressources/ressources.qrc">
        <normaloff>:/img/icons/report</normaloff>:/img/icons/report</iconset>
      </attribute>
      <attribute name="title">
       <string>Experiments</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="6">
        <widget class="QTreeWidget" name="twLocalExperiments">
         <column>
          <property name="text">
           <string>Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Authors</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date of creation</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="4" column="0" colspan="6">
        <widget class="QTreeWidget" name="twOnlineExperiments">
         <column>
          <property name="text">
           <string>Id</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Authors</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date of creation</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Local : ./ressources/</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="6">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Online : genecraft-project.org</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="pbOpenExperimentsFolder">
         <property name="text">
          <string>Open folder</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/experiment_browse</normaloff>:/img/icons/experiment_browse</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pbShareExp">
         <property name="text">
          <string>Share</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/down</normaloff>:/img/icons/down</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pbSaveExp">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/up</normaloff>:/img/icons/up</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pbLoadLocalExp">
         <property name="text">
          <string>Load</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/experiment_load</normaloff>:/img/icons/experiment_load</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pbLoadOnlineExp">
         <property name="text">
          <string>Load</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/experiment_load</normaloff>:/img/icons/experiment_load</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QPushButton" name="pbRefreshOnlineExp">
         <property name="text">
          <string>Refresh</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/refresh</normaloff>:/img/icons/refresh</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="pbRefreshLocalExp">
         <property name="text">
          <string>Refresh</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/refresh</normaloff>:/img/icons/refresh</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="icon">
       <iconset resource="../../../ressources/ressources.qrc">
        <normaloff>:/img/icons/world</normaloff>:/img/icons/world</iconset>
      </attribute>
      <attribute name="title">
       <string>Worlds</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="icon">
       <iconset resource="../../../ressources/ressources.qrc">
        <normaloff>:/img/icons/entity</normaloff>:/img/icons/entity</iconset>
      </attribute>
      <attribute name="title">
       <string>Creatures</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="6">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Local : ./ressources/</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QPushButton" name="pbOpenEntitiesFolder">
         <property name="text">
          <string>Open folder</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/experiment_browse</normaloff>:/img/icons/experiment_browse</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="8">
        <widget class="QTreeWidget" name="twLocalEntities">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Family</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Generation</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="3" column="0" colspan="8">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Online : genecraft-project.org</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="8">
        <widget class="QTreeWidget" name="twOnlineEntities">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Family</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Generation</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="6">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="5">
        <widget class="QPushButton" name="pbSaveEntity">
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/up</normaloff>:/img/icons/up</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QPushButton" name="pbShareEntity">
         <property name="text">
          <string>Share</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/down</normaloff>:/img/icons/down</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pbLoadLocalEntity">
         <property name="text">
          <string>Load</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/entity_add</normaloff>:/img/icons/entity_add</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="7">
        <widget class="QPushButton" name="pbRefreshLocalEntity">
         <property name="text">
          <string>Refresh</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/refresh</normaloff>:/img/icons/refresh</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pbLoadOnlineEntity">
         <property name="text">
          <string>Load</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/entity_add</normaloff>:/img/icons/entity_add</iconset>
         </property>
        </widget>
       </item>
       <item row="5" column="7">
        <widget class="QPushButton" name="pbRefreshOnlineEntity">
         <property name="text">
          <string>Refresh</string>
         </property>
         <property name="icon">
          <iconset resource="../../../ressources/ressources.qrc">
           <normaloff>:/img/icons/refresh</normaloff>:/img/icons/refresh</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../ressources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
